common/libs/keystore.js

```diff
@@ -10,6 +10,13 @@ import { sha3, privateToAddress } from 'ethereumjs-util';
 import scrypt from 'scryptsy';
 import uuid from 'uuid';
 
+export type UtcKeystore = {
+  version: number,
+  id: string,
+  address: string,
+  Crypto: Object
+};
+
```

Comment:
Is a single layer deep sufficient for our purposes, or should all of `Crypto` be typed out?